site/profile/templates/provisioning/bootstrap.sh.erb

```diff
@@ -26,6 +30,7 @@ while getopts ":u:k:r:h" opt; do
       echo "Options:"
       echo "  * Email address: -u <email address>"
       echo "  * Setup key:     -k <setup key>"
+      echo "  * Hostname:      -n <unique hostname>"
```

Comment:
Can you change this to say "Machine name" or something like it?

I don't want there to be confusion about the fact that this is an "alternate" certname that's only used to identify reports on classroom.puppet.com. Calling it hostname might make people think that it's going to have some effect on the machine itself.

That said, I wonder if what we ought to do is just increase the timestamp granularity to include the time-of-day?